src/Sulu/Bundle/AdminBundle/Resources/js/components/RectangleSelection/RectangleSelection.js

```diff
@@ -1,91 +1,149 @@
 import SizeNormalizer from './normalizers/SizeNormalizer';
+import withPercentageValues from './withPercentageValues';
 import rectangleSelectionStyles from './rectangleSelection.scss';
 
 type Props = {
+    backdrop: boolean,
     children?: Node,
     containerHeight: number,
     containerWidth: number,
-    minHeight?: number,
-    minWidth?: number,
+    disabled: boolean,
+    label?: string,
+    minHeight: number | typeof undefined,
+    minSizeNotification: boolean,
+    minWidth: number | typeof undefined,
     onChange: (s: ?SelectionData) => void,
+    onFinish?: () => void,
     round: boolean,
+    usePercentageValues: boolean,
     value: SelectionData | typeof undefined,
 };
 
 @observer
-class RectangleSelection extends React.Component<Props> {
+class RectangleSelectionComponent extends React.Component<Props> {
     static defaultProps = {
+        backdrop: true,
+        disabled: false,
+        minHeight: undefined,
+        minSizeNotification: true,
+        minWidth: undefined,
         round: true,
+        usePercentageValues: false,
+    };
+
+    @computed get value() {
+        const {value} = this.props;
+
+        if (!value) {
+            return this.maximumSelection;
+        }
+
+        return value;
+    }
+
+    componentDidMount() {
+        this.setInitialValue();
+    }
+
+    setInitialValue = () => {
+        const {containerHeight, containerWidth, onChange, value} = this.props;
+
+        if (!containerHeight || !containerWidth) {
+            return;
+        }
+
+        if (!value) {
+            onChange(this.value);
+        }
     };
 
     static createNormalizers(props: Props): Array<Normalizer> {
-        if (!props.containerWidth || !props.containerHeight) {
+        const {
+            containerWidth,
+            containerHeight,
+            minWidth,
+            minHeight,
+            round,
+        } = props;
+
+        if (!containerWidth || !containerHeight) {
             return [];
         }
 
-        const normalizers = [];
-
-        normalizers.push(new SizeNormalizer(
-            props.containerWidth,
-            props.containerHeight,
-            props.minWidth,
-            props.minHeight
-        ));
-
-        if (props.minWidth && props.minHeight) {
-            normalizers.push(new RatioNormalizer(
-                props.containerWidth,
-                props.containerHeight,
-                props.minWidth,
-                props.minHeight
-            ));
-        }
+        const normalizers = [
+            new SizeNormalizer(
+                containerWidth,
+                containerHeight,
+                minWidth,
+                minHeight
+            ),
+            new PositionNormalizer(
+                containerWidth,
+                containerHeight
+            ),
+        ];
 
-        normalizers.push(new PositionNormalizer(props.containerWidth, props.containerHeight));
+        if (minWidth && minHeight) {
```

Comment:
Note to myself: Seems like the order of the normalizers changed, maybe that's the reason the selection is faster than the cursor on my machine now?

Reply:
Nope, also doesn't work that way... But it only happens on Chrome, not on Firefox...

Reply:
Maybe this is only an issue on chrome in linux with a retina display... I am resolving that for now.